### PR TITLE
fix: Actions now typed

### PR DESCRIPTION
**Pull request by wsdt**
## Overview

Typing the Gateway, trying to keep the PRs as tiny as possible. 
This PR types the Actions (will further type them out, but this is just to get started).

## Changes
- Removed orphaned code
- Typed actions

### CODE REVIEW

File: src/containers/modals/walletSelector/WalletSelectorModal.js [hunk 1]
```diff
@@ -19,13 +20,12 @@ import ArrowIcon from 'assets/images/icons/arrowright.svg'
...
 
 import { useWalletConnect } from 'hooks/useWalletConnect'
-import { setConnect } from 'actions/setupAction'
 import styled from 'styled-components'
 
 const StyledSvg = styled(Svg)`
   display: flex;
   svg {
-    fill: ${({ theme }) => theme.name == 'light' ? theme.colors.gray[ 600 ] : theme.colors.gray[ 100 ]}
+    fill: ${({ theme }) => theme.name === 'light' ? theme.colors.gray[ 600 ] : theme.colors.gray[ 100 ]}
```
Comment (sk-enya) [inline src/containers/modals/walletSelector/WalletSelectorModal.js:28]: this will raise warning in esliting.

**Review by sk-enya — APPROVED** [review]
Looks good, please fix the conflict before merging!

File: src/containers/modals/walletSelector/WalletSelectorModal.js (hunk 1 above)
Comment (wsdt) [reply src/containers/modals/walletSelector/WalletSelectorModal.js:28]: I got a warning without it. Will check thanks!

Comment (wsdt) [reply src/containers/modals/walletSelector/WalletSelectorModal.js]: removed thx!

**Pull request merged**